package main.exception;

/**
 * будем кидать, когда нужно прервать текущую операцию и выйти из приложения
 */

public class InterruptOperationException extends Exception {
}
